apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/tally-template-a/tally-template-a.component.ts

```diff
@@ -0,0 +1,118 @@
+import { CustomTemplateResponse } from '../../../../../../models/api-models/Invoice';
+import { TemplateContentUISectionVisibility } from '../../../../../../services/invoice.ui.data.service';
+import * as dayjs from 'dayjs';
+import { GIDDH_DATE_FORMAT, GIDDH_DATE_FORMAT_DD_MM_YYYY } from 'apps/web-giddh/src/app/shared/helpers/defaultDateFormat';
+
+@Component({
+    selector: 'tally-template-a',
+    templateUrl: './tally-template-a.component.html',
+    styleUrls: ['./tally-template-a.component.scss']
+})
+
+export class TallyTemplateAComponent implements OnInit, OnDestroy {
+    /** Holds fields and visibility object */
+    @Input() public fieldsAndVisibility: any = null;
+    /** Holds true if preview mode */
+    @Input() public isPreviewMode: boolean = false;
+    /** Holds true if show company logo */
+    @Input() public showLogo: boolean = true;
+    /** Holds true if show company name */
+    @Input() public showCompanyName: boolean;
+    /** Holds company GSTIN  number as string */
+    @Input() public companyGSTIN: string;
+    /** Holds true if company PAN number as string */
+    @Input() public companyPAN: string;
+    /** Holds template input data */
+    @Input() public inputTemplate: CustomTemplateResponse = new CustomTemplateResponse();
+    /** Holds uploaded company logo source */
+    @Input() public logoSrc: string;
+    /** Holds uploaded image signature source */
+    @Input() public imageSignatureSrc: string;
+    /** Holds true show image signature */
+    @Input() public showImageSignature: boolean;
+    /* This will hold active company*/
+    @Input() public activeCompany: any;
+    /** Holds template UI Section Visibility  status and label name */
+    @Input() public templateUISectionVisibility: TemplateContentUISectionVisibility = new TemplateContentUISectionVisibility();
+    /* This will hold the value if Gst Composition will show/hide */
+    @Input() public showGstComposition: boolean = false;
+    /** Holds voucher type */
+    @Input() public voucherType: string;
+    /** Emits selected section name to edit and show respective options */
+    @Output() public sectionName: EventEmitter<string> = new EventEmitter();
+    /** Holds company setting */
+    public companySetting$: Observable<any> = observableOf(null);
+    /** Holds company address */
+    public companyAddress: string = '';
+    /** Observable to unsubscribe all the store listeners to avoid memory leaks */
+    private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
+    /** Holds dollar symbol */
+    public dollarSymbol = '$';
+    /** Holds true if company Base Currency is Rupee */
+    public isBaseCurrencyRupee = true;
+    /** Holds rupee symbol */
+    public rupeeSymbol = '&#8377';
+
+    constructor(
+        private store: Store<AppState>,
+        private settingsProfileActions: SettingsProfileActions
+    ) {
+        this.companySetting$ = this.store.pipe(select(s => s.settings.profile), takeUntil(this.destroyed$));
```

Comment:
do not use single character, please check in over all file

Reply:
Done